qvm-tests.asd

```diff
@@ -6,8 +6,7 @@
   :description "Regression tests for the QVM."
   :author "Robert Smith <robert@rigetti.com>"
   :license "Apache License 2.0 (See LICENSE.txt)"
-  :depends-on (#:cl-quil
```

Comment:
why is CL-QUIL being taken out of all of these depends-on? Doesn't the test suite explicitly depend on it, even if it's transitively loaded?

Reply:
Good point, I've restored transitive dependencies.